source/GameInterface/GameInterface.csproj

```
@@ -218,15 +229,49 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
     <PackageReference Include="Lib.Harmony" Version="2.3.5" />
-    <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="9.0.2" />
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="9.0.3" />
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="4.13.0" />
     <PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
+    <PackageReference Include="Scriban" Version="6.0.0" />
     <PackageReference Include="Scrutor" Version="6.0.1" />
   </ItemGroup>
     <ItemGroup>
 	    <AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleTo">
 		    <_Parameter1>E2E.Tests</_Parameter1>
 	    </AssemblyAttribute>
     </ItemGroup>
+    <ItemGroup>
+      <EmbeddedResource Include="DynamicSync\Templates\DynamicAssemblyInfoTemplate" />
+      <EmbeddedResource Include="DynamicSync\Templates\Patches\DynamicPatchTemplate">
+        <CopyToOutputDirectory>Never</CopyToOutputDirectory>
+      </EmbeddedResource>
```

Make this one like the rest

```suggestion
      <EmbeddedResource Include="DynamicSync\Templates\Patches\DynamicPatchTemplate" />
```